### PR TITLE
Adds serialization for CancelAfter and FinishAfter

### PR DESCRIPTION
**Pull request by tedkalaw**
## High Level Overview of Change

Provides serialization for CancelAfter and FinishAfter protocol buffers. 

### Context of Change

Each protocol buffer (`Foo`) maps to an equivalent `FooJSON` in `Serializer`. This PR wires this conversion for `CancelAfter` + `FinishAfter`, and adds associated unit tests. 

### Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Refactor (non-breaking change that only restructures code)
- [ ] Tests (You added tests for code that already exists, or your new feature included in this PR)
- [ ] Documentation Updates
- [ ] Release

## Before / After

N/A

## Test Plan

CI - new tests provided.

### CODE REVIEW

File: test/XRP/serializer.test.ts [hunk 1]
```diff
@@ -1761,4 +1762,30 @@ describe('serializer', function (): void {
...
+    const serialized = Serializer.cancelAfterToJSON(cancelAfter)
+
+    // THEN the result is as expected.
+    assert.equal(serialized, cancelAfterTime)
+  })
+
+  it('Serializes a FinishAfter', function (): void {
+    // GIVEN a FinishAfter.
+    const finishAfterTime = 5331715585
+    const finishAfter = new CancelAfter()
```
Comment (amiecorso) [inline test/XRP/serializer.test.ts:1782]:
```suggestion
    const finishAfter = new FinishAfter()
```

File: test/XRP/serializer.test.ts [hunk 2]
```diff
@@ -1761,4 +1762,30 @@ describe('serializer', function (): void {
...
+    finishAfter.setValue(finishAfterTime)
+
+    // WHEN it is serialized.
+    const serialized = Serializer.cancelAfterToJSON(finishAfter)
```
Comment (amiecorso) [inline test/XRP/serializer.test.ts:1786]:
```suggestion
    const serialized = Serializer.finishAfterToJSON(finishAfter)
```

**Comment by tedkalaw** [comment]
l

File: test/XRP/serializer.test.ts (hunk 1 above)
Comment (tedkalaw) [reply test/XRP/serializer.test.ts]: got it

File: test/XRP/serializer.test.ts (hunk 2 above)
Comment (tedkalaw) [reply test/XRP/serializer.test.ts]: got it

**Pull request merged**